ivadomed/loader/utils.py

```diff
@@ -13,7 +13,6 @@
 import torch
 from loguru import logger
 from sklearn.model_selection import train_test_split
-from torch._six import string_classes
```

Comment:
Our first bit of version incompatibility: We were accessing `torch._six`, which is a now-removed internal API.

The fix for this is simple enough, though -- we can follow `torchvision`'s example here: https://github.com/pytorch/vision/pull/7243

Reply:
Resolved!